doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows the updating of both the application core and the network core in one go.
+
+To enable the simultaneous update of multiple images, set the following options:
```

Comment:
Should we mention that they need to be set for MCUboot config ?

Reply:
Reworded some things. Should be better now.